<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMenuBar</class>
 <widget class="QWidget" name="CMenuBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>33</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QToolButton" name="toolButtonMenuIcon">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../resource.qrc">
       <normaloff>:/images/resource/images/configurator.ico</normaloff>:/images/resource/images/configurator.ico</iconset>
     </property>
     <property name="popupMode">
      <enum>QToolButton::InstantPopup</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonMinimizeTabMenu">
     <property name="minimumSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Другие кнопки</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../resource.qrc">
       <normaloff>:/images/resource/images/down_indicator.png</normaloff>:/images/resource/images/down_indicator.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>7</width>
       <height>7</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Ctrl+S</string>
     </property>
     <property name="popupMode">
      <enum>QToolButton::InstantPopup</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelTitleProgram">
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>725</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonMinimizeWindow">
     <property name="icon">
      <iconset resource="../../resource.qrc">
       <normaloff>:/images/resource/images/minimize.png</normaloff>:/images/resource/images/minimize.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonExpandWindow">
     <property name="icon">
      <iconset resource="../../resource.qrc">
       <normaloff>:/images/resource/images/expanded.png</normaloff>:/images/resource/images/expanded.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonCloseWindow">
     <property name="icon">
      <iconset resource="../../resource.qrc">
       <normaloff>:/images/resource/images/close.png</normaloff>:/images/resource/images/close.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
